xknx/management/procedures.py

```diff
@@ -21,6 +23,26 @@
 logger = logging.getLogger("xknx.management.procedures")
 
 
+class RestartEraseCode(Enum):
+    """Enum class for Confirmed Restart Erase Code."""
+
+    CONFIRMED_RESTART = b"\x01"
+    FACTORY_RESET = b"\x02"
+    RESET_IA = b"\x03"
+    RESET_AP = b"\x04"
+    RESET_PARAM = b"\x05"
+    RESET_LINKS = b"\x06"
+    FACTORY_RESET_WITHOUT_IA = b"\x07"
```

Comment:
Consider using integers like in the specifications (eg. `CONFIRMED_RESTART = 0x01`)
Maybe move to `xknx/telegram/apci_const.py` or `xknx/management/const.py` something like that.